bash_scripts/bash_script/script.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+ETC_HOSTS=/etc/hosts
+IFS=$'\n'
+
+function dirs_older {
+        logger "dirs older"
+        date=$(date +"%b %d %H:%M" -d "$1 day ago")
```

Comment:
Better to name all variables:
```
PATH_TO_SEARCH=$1
FILES_AGE=$2
....
```
and then use variables instead of $N

Reply:
use variables insted $1, $2